styles.css

```diff
@@ -388,10 +395,365 @@
+    width: 100%;
+    background-color: #1a2236;
+    height: 25rem;
+  }
+
+  .big-laptop img {
+    width: 100%;
+    height: 13rem;
+  }
+
+  .work {
+    display: none;
+  }
+
+  .work-desktop {
+    display: block;
+  }
+
+  .grid-container1 {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: auto;
+  }
+
+  .grid1 {
+    height: 16rem;
+    width: 29rem;
+    position: relative;
+    bottom: -6.5rem;
+  }
+
+  .grid2 {
+    width: 32.5rem;
+  }
+
+  .grid3 {
+    padding-left: 1.25rem;
+    background-color: #1a2236;
+    width: 100%;
+    position: relative;
+    bottom: -3.25rem;
+  }
+
+  .grid3 img {
+    position: relative;
+    width: 100%;
+    height: 80%;
+    bottom: -2.5rem;
+  }
+
+  .grid3 button {
+    background-color: white;
+    color: #1a2236;
+    width: 11.125rem;
+    height: 2.75rem;
+    line-height: 1.5rem;
+    font-family: "Poppins", sans-serif;
+    font-weight: 500;
+    font-size: 18px;
+    border: 1px solid #1a2236;
+  }
+
+  .About button {
+    width: 15.875rem;
+  }
+
+  .grid3 button:hover {
+    background-color: #1a2236;
+    color: white;
+    border: 1px solid white;
+  }
+
+  .grid3 button:active {
+    border: 2px solid white;
+  }
+
+  .grid-container2 {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    grid-template-rows: 70% 30%;
+    margin-bottom: 10rem;
+  }
+
+  .grid4 {
+    width: 100%;
+    height: 100%;
+  }
+
+  .grid5 {
+    position: relative;
+    bottom: -9.5rem;
+  }
+
+  .projects button:hover {
+    background-color: #1a2236;
+    color: white;
+    border: 1px solid white;
+  }
+
+  .projects button:active {
+    border: 2px solid white;
+  }
+
+  /* Desktop About */
+
+  .About {
+    display: flex;
+    flex-direction: row-reverse;
+    gap: 6rem;
+    margin-bottom: 10rem;
+  }
+
+  .About-container {
+    margin-right: 6rem;
+  }
+
+  .knowledge {
+    display: grid;
+    grid-template-columns: 18.063rem 18.063rem;
+    grid-template-rows: 18.063rem 18.063rem;
+    margin-left: 10rem;
+  }
+
+  .grid-picture {
+    display: block;
+    position: relative;
+  }
+
+  .grid-picture img {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+  }
+
+  .About ul {
+    padding: 0;
+  }
+
+  .About h2 {
+    font-size: 6.875rem;
+    margin: 1.875rem auto;
+  }
+
+  .get-contact {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding-left: 20rem;
+    position: relative;
+  }
+
+  .get-contact h2 {
+    font-size: 6.875rem;
+    margin-bottom: 2.5rem;
+  }
+
+  form {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 2.188rem;
+  }
+
+  #name {
+    width: 20rem;
+    padding: 18px 18px;
+  }
+
+  #email {
+    width: 20rem;
+    padding: 18px 18px;
+  }
+
+  #comment-box {
+    width: 42.188rem;
+  }
+
+  .get-contact p {
+    letter-spacing: -1;
+    margin-bottom: 2.5rem;
+    width: 70%;
+  }
+
+  .container-button {
+    margin: auto 13rem;
+    margin-bottom: 10rem;
+  }
+
```

Comment:
- [ ] In this section the button margins are different from the design, kindly stick to the Figma design.
![rev4](https://user-images.githubusercontent.com/53347966/230516834-acbf483d-b961-4adb-bca1-83030d9acb73.JPG)